### DISCUSSION TITLE
What are some of the reasons why authors archieve a repository?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [community/community#23076] [New to GitHub]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/b/b/bbf3d690e9bca5795f17d94e0c6ca7f5484c9e47.png" data-download-href="/uploads/short-url/qOHHSM7HqvvK4SejVzwLM7kU8Bh.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/12727582/181074710-eea2dea3-df6a-4cac-954d-884c54f40888.png" alt="image" data-base62-sha1="qOHHSM7HqvvK4SejVzwLM7kU8Bh" width="690" height="348" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/b/b/bbf3d690e9bca5795f17d94e0c6ca7f5484c9e47_2_690x348.png, https://aws1.discourse-cdn.com/github/optimized/3X/b/b/bbf3d690e9bca5795f17d94e0c6ca7f5484c9e47_2_1035x522.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/b/b/bbf3d690e9bca5795f17d94e0c6ca7f5484c9e47_2_1380x696.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/b/b/bbf3d690e9bca5795f17d94e0c6ca7f5484c9e47_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×969 109 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>why authors archieve a repository?</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>when you look at the README, they have mentioned that, it’s already retired, it’s just a way to show that it’s no longer being maintained, maybe not the exact project but the repo itself</p>

**Comment by airtower-luna** [comment]
<p>If you mean in general, archiving shows the repository isn’t being worked on any more for whatever reason (e.g. a project moved or was stopped). If a repository is archived people can clearly see that, but they can still see the data if they want to.</p>

**Comment by queenofcorgis** [comment]
<p>Thanks <a class="mention" href="/jdevstatic">@jdevstatic</a> and <a class="mention" href="/airtower-luna">@airtower-luna</a> for chiming in here :heart_eyes_cat:</p>
